### ISSUE TITLE
vim-scripts is no longer supported by vim-plug

### ISSUE THREAD

**Issue by sedrubal** [fisadev/fisa-nvim-config#3]
Because of https://github.com/junegunn/vim-plug/issues/625 [`vim-plug`](https://github.com/junegunn/vim-plug) won't support http://vim-scripts.org anymore. But in the config there are still the commands `Plug "IndexedSearch"` and `Plug "YankRing"` which results in this warning when nvim starts:

```
[vim-plug] Invalid argument: IndexedSearch (implicit `vim-scripts' expansion is deprecated)
[vim-plug] Invalid argument: YankRing.vim (implicit `vim-scripts' expansion is deprecated)
```

**Comment by fisadev** [comment]
True, thanks for bringing this up. I've fixed it and included it in the new beta :) (v9.5)

**Comment by sedrubal** [comment]
Thanks 🎉
